src/test/java/com/github/javafaker/CPFTest.java

```diff
@@ -0,0 +1,47 @@
+package com.github.javafaker;
+
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static com.github.javafaker.matchers.MatchesRegularExpression.matchesRegularExpression;
+
+import com.github.javafaker.shared.CPFUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class CPFTest extends AbstractFakerTest {
+
+    private String validCPF;
+    private String invalidCPF;
+
+    @Before
+    public void setup() {
+        validCPF = faker.cpf().valid();
+        invalidCPF = faker.cpf().invalid();
+    }
+
+    @Test
+    public void isValidCPF() {
+        assertTrue(isCPFValid(validCPF));
+    }
+
+    @Test
+    public void isInvalidCPF() {
+        assertFalse(isCPFValid(invalidCPF));
+    }
+
+    @Test
+    public void formattedCPF() {
+        assertThat(validCPF, matchesRegularExpression("(^\\d{3}\\x2E\\d{3}\\x2E\\d{3}\\x2D\\d{2}$)"));
+    }
+
+    private static Boolean isCPFValid(String cpf) {
```

Comment:
@eliasnogueira any chance you could put some documentation in there? Thanks!

Reply:
Hey @codingricky I added some information. If it's not ok, please give me a feedback :)